app/controllers/member/dashboard_controller.rb

```diff
@@ -2,6 +2,8 @@ class Member::DashboardController < Member::BaseController
 
   def index
     authorize!(:show, :member_dashboard)
+    authorize!(:index, List)
```

Comment:
Well spotted. This is authorising against the entire `List` model, whereas in the other actions I am authorising against the instance of `@lists`. I had done it this way before because I was returning the collection of ALL the lists. In subsequent changes I have since only returned a collection of lists where the list belongs to the user, so in this case I need to change the authorisation to be against the instance rather than the entire model. :+1: